nodejs/week1/avg.js

```diff
@@ -0,0 +1,17 @@
+
+const arg = process.argv.slice(2);
+
+const numbers = arg.map(Number).filter(num => !isNaN(num));
```

Comment:
I see you decided to tolerate and ignore the arguments, which are not numbers! 👍